templates/filebeat-6.yml

```diff
@@ -133,3 +133,9 @@ output:
 {% if juju_principal_unit -%}
 name: {{ juju_principal_unit }}
 {%- endif %}
+seccomp:
+  default_action: errno
+  syscalls:
+    - action: allow
+      names:
+        - clone3
```

Comment:
As we already supports filebeat 7, could you please check if this syscall is also needed for `filebeat-7.yaml`? You can test it by replacing the default value of [`install_sources`](https://charmhub.io/filebeat/configure#install_sources) config option with `"deb https://artifacts.elastic.co/packages/7.x/apt stable main"`

Reply:
Yes, I wanted to mention this as well. Filebeat7 has this problem fixed in [https://www.elastic.co/guide/en/beats/libbeat/master/release-notes-7.15.2.html](7.15.2)